magda-web-client/src/Components/Home/Stories.js

```diff
@@ -108,12 +108,14 @@ class Stories extends Component {
                                         case 2:
                                         case 3:
                                             return (
-                                                <div className="stories-container">
+                                                <div
+                                                    className="stories-container"
+                                                    key={r}
```

Comment:
`r` seems the row number. thus appropriate to be the key (as you could have two elements with the same key)?

Reply:
The `div` above has the key `r`. This is another list of `div`s within that row. If the key is `r` then all of the `div`s in this row will have the same key.

Reply:
I had not looked into the code but if we are sure the row number is only used on row level div then it should be OK.
Also:
![image](https://user-images.githubusercontent.com/674387/69025918-fc829600-0a1c-11ea-8391-cf11a50ea3d7.png)
seems still some divs have no key?
Moreover, I suspect the no `key props` warning should be generated for the mobile view ---- not the code block you modify here.